src/recognizerjs/recognizer-constructor.js

```diff
@@ -54,6 +54,8 @@ import stateStr from './state-str';
  */
 export default class Recognizer {
   constructor(options) {
+    Recognizer.prototype.defaults = {};
```

Comment:
modifying the prototype shouldn't be done from within the constructor, either instance properties are set in the constructor or we add to the prototype after we build the class.